test/mock_batch_test.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore_mocks/cloud_firestore_mocks.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() async {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  String users = "users";
+  MockFirestoreInstance firestore = MockFirestoreInstance();
+
+  Map<String, dynamic> userDocumentMock(String userId) {
+    Map<String, dynamic> data = {
+      "id": userId,
+      "updatedAt": DateTime.now(),
+      "createdAt": DateTime.now(),
+    };
+    return data;
+  }
+
+  Future<List<Map<String, dynamic>>> createUsers({int times}) async {
+    List<Map<String, dynamic>> userDataList = [];
+    for (int i = 1; i <= times; ++i) {
```

Comment:
Even though this loop is absolutely correct, would you mind writing it as `for (var i = 0; i < times; i++)` for consistency with the rest of the code? This is also not consistent with the loops you wrote at lines 35 and 66. :)